Header header
int8 proximity_1
int8 proximity_2
int8 proximity_3
